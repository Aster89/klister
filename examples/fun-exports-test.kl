#lang "n-ary-app.kl"
(import (shift "n-ary-app.kl" 1))
(import (shift "n-ary-app.kl" 2))
(import "fun-exports.kl")

(example a)
(meta (example b))
(meta (meta (example the-a)))
