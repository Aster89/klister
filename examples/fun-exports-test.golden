Signal 1 : Signal
Signal 2 : Signal
Signal 3 : Signal
